scripts/release_canary.sh

```diff
@@ -4,11 +4,11 @@ set -e
 
 : "${NPM_TOKEN?Required env variable NPM_TOKEN}"
 
+COMMIT_MESSAGE=$(git log --format=oneline -n 1 $CIRCLE_SHA1))
+
 # Only trigger the canary release when:
-# - the branch is `master`
-# - the build has not been triggered by a pull request
 # - the commit message does not contain `[skip publish]`
-if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ] && [[ ! "$TRAVIS_COMMIT_MESSAGE" =~ \[skip\ publish\] ]]; then
+if [[ ! "$COMMIT_MESSAGE" =~ \[skip\ publish\] ]]; then
```

Comment:
The job is only triggered on `master` (as CircleCI's config is more powerfull). If it is `master` it also can't be a pull request.